webpack/webpack.node.config.js

```diff
@@ -0,0 +1,11 @@
+const webpack = require('webpack');
+
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
Is this used anywhere?

Reply:
Ah never mind, I assume we’ll use this in the future to minify the code?

Reply:
Nah, this is some unused code, removing..

Reply:
Removed.